src/editors/activeDBCodeLensProvider.ts

```diff
@@ -0,0 +1,57 @@
+import { URLSearchParams } from 'url';
+import * as vscode from 'vscode';
+
+export default class ActiveDBCodeLensProvider implements vscode.CodeLensProvider {
+  private _codeLenses: vscode.CodeLens[] = [];
+  private _connectionController: any;
+  private _runtime: any;
+  private _activeDB?: string;
+  private _onDidChangeCodeLenses: vscode.EventEmitter<
+    void
+  > = new vscode.EventEmitter<void>();
+  public readonly onDidChangeCodeLenses: vscode.Event<void> = this
+    ._onDidChangeCodeLenses.event;
+
+  constructor(connectionController?: any) {
+    this._connectionController = connectionController;
+
+    vscode.workspace.onDidChangeConfiguration(() => {
+      this._onDidChangeCodeLenses.fire();
+    });
+  }
+
+  public setActiveDB(activeDB: string): void {
+    this._activeDB = activeDB;
+    this._onDidChangeCodeLenses.fire();
+  }
+
+  public provideCodeLenses(): vscode.CodeLens[] {
+    const activeConnection = this._connectionController.getActiveDataService();
+
+    if (!activeConnection) {
+      return [];
+    }
+
+    this._codeLenses = [new vscode.CodeLens(new vscode.Range(
+      new vscode.Position(0, 0),
+      new vscode.Position(0, 0),
+    ))];
+
+    return this._codeLenses;
+  }
+
+  public resolveCodeLens?(codeLens: vscode.CodeLens): vscode.CodeLens {
+    if (this._runtime) {
+      this._activeDB = this._runtime.openContextRuntime.serviceProvider.nodeTransport.mongoClient.s.options.dbName;
+    }
+
+    codeLens.command = {
+      title: `Active db is ${this._activeDB}`,
+      tooltip: "Tooltip provided by Active DB CodeLens Provider",
+      command: "mdb.runDBHelpInPlayground",
+      arguments: ["Argument 1", { connectionInstanceId: '111111' }]
```

Comment:
I think we can get rid of the arguments here since we aren't using them them. When I hover I can't see the tooltip - is there a way to show a tooltip? Maybe it could read `Click to show information on db functions.`. Trying out clicking on this I think the behavior might not be totally intuitive for the user - it might hide something they were looking at in the output window - I like the idea of giving the user some help though.

Reply:
I was playing with configuration and didn't delete it. Yeah, we don't need it here at all. This PR is open for toooo long :)

Reply:
I will check how to do a tooltip.

Reply:
I didn't find a way how to add a tooltip to code lens. It seems that the text we display there is just a string and codeLens can't be extensible at the moment. And I use this `db.help()` command because the command is required prop. Without a command, we can't specify a text.